### PR TITLE
Scheduler readme documentation

### PR DESCRIPTION
**Pull request by Thurfjell**
### Ticket:

- [16951](https://dev.azure.com/OGCIO-Digital-Services/Digital%20Services%20Programme/_workitems/edit/16951)

### Description

Add a description what the scheduler api does and some examples how to use it.

## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [ ] **New feature**
- [ ] **Dev change**

### Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works

### Screenshots:

N/A

### CODE REVIEW

File: apps/scheduler-api/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+Recall same endpoint to simulate a cron like behaviour
+
+```javascript
+someApp.Post(
+  "your.client.handler/jobs/:id",
+  async function jobHandler(req, res) {
+    const id = req.params.id;
+    await logic(id);
+    fetch(...same original call with execute at now() + arbitrary unit of time)
```
Comment (peschina) [inline apps/scheduler-api/README.md:74]: I'm not sure whether this pseudo code is better or worse than just a comment. Maybe worth seeing what the others think as well

**Review by peschina — APPROVED** [review]
LGTM!

File: apps/scheduler-api/README.md (hunk 1 above)
Comment (edge33) [reply apps/scheduler-api/README.md:74]: I would prefer to have sample with the actual fetch url.

File: apps/scheduler-api/README.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+someApp.Post(
+  "your.client.handler/jobs/:id",
+  async function jobHandler(req, res) {
+    const id = req.params.id;
+    void doSomethingReallyLengthy(id);
+  },
+);
+```
+
+Recall same endpoint after a day, to simulate a cron like behaviour
```
Comment (Thurfjell) [inline apps/scheduler-api/README.md:66]:
```suggestion
Recall same endpoint after an interval, to simulate a cron like behaviour
```

**Comment by edge33** [comment]
LGTM

**Review by edge33 — APPROVED** [review]

**Pull request merged**